ts/packages/ts/src/commands/UpsertCommand.ts

```diff
@@ -66,6 +67,25 @@ export class UpsertCommand extends Command<void> {
             }),
           );
         }
+        if (value && cfgu.aliases) {
+          if (AllAliases.some((a) => a === key)) {
+            throw new Error(
+              ERR(`invalid config  key "${key}"`, {
+                location: scopeLocation,
+                suggestion: `upserting more than one of any of the above is invalid.`,
+              }),
+            );
+          }
+          if (cfgu.aliases.some((a) => a === key)) {
```

Comment:
This check's logic is opposite to the check your'e doing in EvalCommand line 259.
Can you explain the thought process?

Reply:
for eval i created a function which will iterate result and  if it keys has aliases then it add its alias with it value